src/fontra/client/core/utils.js

```diff
@@ -325,3 +325,7 @@ export function rgbaToCSS(rgba) {
   }
   return `rgb(${channels.join(",")})`;
 }
+
+export function clampedNumber(number, min, max) {
```

Comment:
I think we can call this function `clamp`.

Btw. there is a functionally equavalent function called `minmax` defined elsewhere, and there's some inline use of `Math.min(Math.max(...), ...)` that may be replaced with `clamp()`. I would love you to do that, but after this PR.

Reply:
I see in `cjk-design-frame.js` and 'ui-list.js'.